docs/puml/monte_carlo_simulation.puml

```diff
@@ -0,0 +1,14 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:initialize households;
+:current_time = start_time
+appliance_state_distribution = initial_appliance_state_distribution * nr_appliances;
+note left: number of devices in the different stages \n initially all appliances are off (build up)
+:assign transition matrix;
+note left:
+
+stop

```

Comment:
This UML isn't comprehensive, right? Maybe you can add a step more that says / highlights "rest to be defined".